sw/device/lib/drivers/power_manager/power_manager.c

```diff
@@ -24,12 +23,7 @@ void power_gate_core_asm()
         "sw  a1, %[power_manager_power_gate_core_reg_offset](a0)\n"
 
         // write WAKEUP_STATE[0] = 1
-        "sw  a1, %[power_manager_wakeup_state_reg_offset](a0)\n" : : \
-        \
-        [base_address_20bit] "i" (POWER_MANAGER_START_ADDRESS >> 12), \
-        [power_manager_power_gate_core_reg_offset] "i" (POWER_MANAGER_POWER_GATE_CORE_REG_OFFSET), \
-        [power_manager_wakeup_state_reg_offset] "i" (POWER_MANAGER_WAKEUP_STATE_REG_OFFSET) \
-    );
+        "sw  a1, %[power_manager_wakeup_state_reg_offset](a0)\n" : :                 [base_address_20bit] "i" (POWER_MANAGER_START_ADDRESS >> 12),         [power_manager_power_gate_core_reg_offset] "i" (POWER_MANAGER_POWER_GATE_CORE_REG_OFFSET),         [power_manager_wakeup_state_reg_offset] "i" (POWER_MANAGER_WAKEUP_STATE_REG_OFFSET)     );
```

Comment:
These long lines are note readable, use multiple lines

Reply:
You can use the same syntax I used in the upf template

Reply:
Solved.